pkg/cmd/root.go

```diff
@@ -31,6 +32,7 @@ import (
 	"gopkg.in/gookit/color.v1"
 )
 
+var wg *sync.WaitGroup
```

Comment:
Shouldn't `wg.Wait()` be called at some point?

Reply:
We call `Wait()` at the bottom of this file in `Execute()`

Reply:
Ah, I see now. Nevermind.